spec/system/create_work_deposit_spec.rb

```diff
@@ -65,6 +65,31 @@
     # Filling in abstract
     fill_in('work_abstract', with: abstract_fixture)
 
+    # Clicking on Next to go to dates tab
+    click_link_or_button('Next')
+    expect(page).to have_css('.nav-link.active', text: 'Dates (optional)')
+    expect(page).to have_text('Enter dates related to your deposit')
+
+    # Filling in dates
+    within('fieldset#publication-date') do
```

Comment:
Would you consider:

```suggestion
    within_fieldset('publication-date') do
```

❓